### PR TITLE
DOC-5623: Updated Run a sample CorDapp doc

### PR DESCRIPTION
**Pull request by amiegrace12**
- Updated folder names
- Updated Windows command
- Added relref links
- Fixed formatting issues

Preview built here: https://amie.preview.docs.r3.com/en/platform/corda/4.10/enterprise/cordapps/tutorial-cordapp.html#step-3-deploy-the-cordapp-locally

### CODE REVIEW

File: content/en/platform/corda/4.10/enterprise/cordapps/tutorial-cordapp.md [hunk 1]
```diff
@@ -94,23 +94,19 @@ The project containing the sample CorDapp opens.
...
       * A `runnodes.bat` batch file for running all the nodes simultaneously on Windows
 
-    See **Appendix B** for the node structure.
-
+    See [Appendix B]({{< relref "#appendix-b-node-structure-1" >}}) for the node structure.
 
 {{< note >}}
-`deployNodes` is a utility task that can be used in a development environment to create a new set of nodes for testing a CorDapp. In a production environment, you would create a single node as described in [Creating nodes locally]({{< relref "../node/deploy/generating-a-node.md" >}}) instead, and build your CorDapp JARs as described
-in [Building and installing a CorDapp](cordapp-build-systems.md).
+`deployNodes` is a utility task that can be used in a development environment to create a new set of nodes for testing a CorDapp. In a production environment, you would create a single node as described in [Creating nodes locally]({{< relref "../node/deploy/generating-a-node.md" >}}) instead, and build your CorDapp JARs as described in [Building and installing a CorDapp]({{< relref "cordapp-build-systems.md" >}}).
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/4.10/enterprise/cordapps/tutorial-cordapp.md:109]:
```suggestion
`deployNodes` is a utility task that can be used in a development environment to create a new set of nodes for testing a CorDapp. In a production environment, you would instead create a single node as described in [Creating nodes locally]({{< relref "../node/deploy/generating-a-node.md" >}}), and build your CorDapp JARs as described in [Building and installing a CorDapp]({{< relref "cordapp-build-systems.md" >}}). 
```

File: content/en/platform/corda/4.7/enterprise/cordapps/tutorial-cordapp.md [hunk 2]
```diff
@@ -233,10 +233,10 @@ The first step is to deploy the CorDapp to nodes running locally. To do this:
 1. Open a terminal window in the `cordapp-example` directory.
 2. Run the `deployNodes` Gradle task to build four nodes with the CorDapp installed on them:
       * Unix/Mac OSX: `./gradlew deployNodes`
-      * Windows: `gradlew.bat deployNodes`
-3. After the build finishes, navigate to the `workflows-java/build/nodes` or `workflows-kotlin/build/nodes` folder - you should see the following output:
+      * Windows: `./gradlew.bat deployNodes`
+3. When the build finishes, go to the `samples-java/Basic/cordapp-example/build/nodes` or `samples-kotlin/Basic/cordapp-example/build/nodes` folder. You will see the following output:
       * A folder for each generated node
-      * A `runnodes` shell script for running all the nodes simultaneously on osX
+      * A `runnodes` shell script for running all the nodes simultaneously on OSX
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/4.7/enterprise/cordapps/tutorial-cordapp.md:239]: Should this be "macOS" now? https://en.wikipedia.org/wiki/MacOS

File: content/en/platform/corda/4.8/enterprise/cordapps/tutorial-cordapp.md [hunk 3]
```diff
@@ -94,23 +94,19 @@ The project containing the sample CorDapp opens.
 1. Open the command line from the `cordapp-example` directory.
 2. Run the `deployNodes` Gradle task:
       * Unix/Mac OSX: `./gradlew deployNodes`
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/4.8/enterprise/cordapps/tutorial-cordapp.md:96]: As above, thinks it's now called "macOS".

File: content/en/platform/corda/4.9/enterprise/cordapps/tutorial-cordapp.md [hunk 4]
```diff
@@ -94,23 +94,19 @@ The project containing the sample CorDapp opens.
...
       * A `runnodes.bat` batch file for running all the nodes simultaneously on Windows
 
-    See **Appendix B** for the node structure.
-
+    See [Appendix B]({{< relref "#appendix-b-node-structure-1" >}}) for the node structure.
 
 {{< note >}}
-`deployNodes` is a utility task that can be used in a development environment to create a new set of nodes for testing a CorDapp. In a production environment, you would create a single node as described in [Creating nodes locally]({{< relref "../node/deploy/generating-a-node.md" >}}) instead, and build your CorDapp JARs as described
-in [Building and installing a CorDapp](cordapp-build-systems.md).
+`deployNodes` is a utility task that can be used in a development environment to create a new set of nodes for testing a CorDapp. In a production environment, you would create a single node as described in [Creating nodes locally]({{< relref "../node/deploy/generating-a-node.md" >}}) instead, and build your CorDapp JARs as described in [Building and installing a CorDapp]({{< relref "cordapp-build-systems.md" >}}).
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/4.9/enterprise/cordapps/tutorial-cordapp.md:109]:
```suggestion
`deployNodes` is a utility task that can be used in a development environment to create a new set of nodes for testing a CorDapp. In a production environment, you would instead create a single node as described in [Creating nodes locally]({{< relref "../node/deploy/generating-a-node.md" >}}), and build your CorDapp JARs as described in [Building and installing a CorDapp]({{< relref "cordapp-build-systems.md" >}}).
```

**Review by paulmoloneyr3 — CHANGES_REQUESTED** [review]

**Review by paulmoloneyr3 — APPROVED** [review]

**Review by yarivus — COMMENTED** [review]
thx @amiegrace12

**Pull request merged**